src/main/frontend/handler/editor.cljs

```diff
@@ -3425,20 +3440,26 @@
                  (collapse-block! uuid))))))))))
 
 (defn- collapse-all!
```

Comment:
This function is private too.

Reply:
@andelf Now collapse-all! and expand-all! are public functions